lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -3519,6 +3519,11 @@ used to generate dynamic options.
         <xs:documentation xml:lang="en">Documentation for startswith</xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="meta_file_key" type="xs:string">
```

Comment:
Can you separate out the meta_file_key changes, I assume that's handled in https://github.com/galaxyproject/galaxy/pull/11832 ?

Reply:
It's right. Once the `meta_file_key` implemented in #11832 gets merged into Galaxy, dynamic options can be made using the metadata FileParameter, `hyper_params` specifically, of `H5MLM`.